tests/runtest.cmd

```diff
@@ -356,6 +356,9 @@ echo jitstressregs n    - Runs the tests with COMPlus_JitStressRegs=n
 echo jitminopts         - Runs the tests with COMPlus_JITMinOpts=1
 echo jitforcerelocs     - Runs the tests with COMPlus_ForceRelocs=1
 echo gcstresslevel n    - Runs the tests with COMPlus_GCStress=n
+echo     0: None                                1: GC on transitions to preemptive GC
+echo     2: GC on every allowable JITed instr   4: GC on every allowable JITed instr
```

Comment:
i think you meant here 
 1,    // GC on all allocs and 'easy' places
2,    // GC on transitions to preemptive GC